### ISSUE TITLE
Problem with 512 bits genes

### ISSUE THREAD

**Issue by osmancandogan** [Stiggu/agp-py#2]
Great work!
I think I found a small problem. When dealing with 512 bits hex type genes, the code always raised an error and when I change
"return decoded_hex" in main.py line:33 to return "00"+decoded_hex the code did not raise error and worked correctly.
``` 
Traceback (most recent call last):
  File "agp_py\main.py", line 17, in __init__
    self.genes: dict = self.axie_gene_from_binary(self.bin_dict)
  File "agp_py\main.py", line 65, in axie_gene_from_binary
    "cls": ClassMap[bin_data["cls"]].value,
KeyError: '10100'
```
Also this is my first time submitting an issue, if I did something wrong, sorry about that.

**Comment by Stiggu** [comment]
Hey hey!

Just fixed this on last commit 119f39f, I also have updated the pypi package.

Thank you for the report, I was parsing some 256 genes and basically forgot that 512 genes existed.

I'll let the issue open for a couple of days (or until you confirm that it does indeed work with all of your 512 genes)

**Comment by osmancandogan** [comment]
I just tested for a couple of thousands of genes and worked correctly for me. Thanks for the update!
